html/changelogs/Incoming5643-slimechanges.yml

```diff
@@ -0,0 +1,6 @@
+author: Incoming5643
+
+delete-after: True
+
+changes: 
+  - rscadd: Nanotrasen scientists have recently reported mutations within the pink family line of slimes. Station scientists are encoraged to investigate."
```

Comment:
I don't mean you need to be explicit, but this entry essentially amounts to: "something has changed, can you figure out what?"

I'm just doubtful that from reading this, people would infer that the descriptions had changed.